lib/api/streamer.js

```diff
@@ -8,11 +8,12 @@ var _ = require('underscore');
 var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 
-var QUERY_INTERVAL = 100; // 100ms
+var QUERY_INTERVAL = 2000; // 2s
```

Comment:
@mkuklis @sapraaman Major reason this is 2s and not 1s? The stats bar is 1s so I thought it would be nice to have them the same. In fact perhaps it could even be a constant in the socket handler.